class NotAllowedError(Exception):
    pass


class NotFoundError(Exception):
    pass

